compiler/testData/diagnostics/tests/funInterface/functionDelegateClashOnJvm.fir.kt

```diff
@@ -15,7 +15,7 @@ fun interface F3 {
     fun invoke()
 }
 
-fun interface F4 {
+<!FUN_INTERFACE_WRONG_COUNT_OF_ABSTRACT_MEMBERS!>fun<!> interface F4 {
```

Comment:
Those changes also incorrect. Function `functionDelegate` has default implementation, so `F4` can be `fun interface`. See FE 1.0 counterpart in `functionDelegateClashOnJvm.kt`